README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@bkeepers as a sidenote, why is this getting so much criticism now, both from trolls, concerned people and even academics is because this is a project de facto pursued by Github and other large companies. It is setting a precedent. Hence, it does not really matter that much whether it does or does not include all Github projects. It includes enough.